dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/TrackerTrackedEntityCriteriaMapperTest.java

```diff
@@ -574,4 +574,51 @@ void testMappingOrderParamsGivenInvalidField()
             () -> mapper.map( criteria ) );
         assertEquals( "Invalid order property: invalid", e.getMessage() );
     }
+
+    @Test
+    void shouldCreateAttributeFiltersWhenFilterHasMultipleValuesAndValueHasDelimiter()
+        throws BadRequestException,
+        ForbiddenException
+    {
+        criteria.setFilter( Set.of( TEA_2_UID + ":like:project:x:eq:2" ) );
+        TrackedEntityInstanceQueryParams params = mapper.map( criteria );
+
+        List<QueryFilter> actualFilters = params.getFilters().stream().flatMap( f -> f.getFilters().stream() )
+            .collect( Collectors.toList() );
+
+        assertContainsOnly( List.of(
+            new QueryFilter( QueryOperator.LIKE, "project:x" ),
+            new QueryFilter( QueryOperator.EQ, "2" ) ), actualFilters );
+    }
+
+    @Test
+    void shouldThrowBadRequestWhenFilterHasOperatorInWrongFormat()
+    {
+        criteria.setFilter( Set.of( TEA_1_UID + ":lke:value" ) );
+        BadRequestException exception = assertThrows( BadRequestException.class,
+            () -> mapper.map( criteria ) );
+        assertEquals( "Query item or filter is invalid: " + TEA_1_UID + ":lke:value", exception.getMessage() );
+    }
+
+    @Test
+    void shouldCreateQueryFiltersWhenQueryHasOperatorAndValueWithDelimiter()
+        throws ForbiddenException,
+        BadRequestException
+    {
+        criteria.setQuery( "like:project:x" );
+        TrackedEntityInstanceQueryParams params = mapper.map( criteria );
```

Comment:
dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/RequestParamUtils.java has dedicated tests for `parseQueryItem`. Should we move some of these over there so that we do not duplicate tests? And maybe only keep one test in the Mappers to test the integration between `RequestParamUtils` and the mappers.